src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.cpp

```diff
@@ -139,7 +137,7 @@ void WebRTCTransportProviderServer::InvokeCommand(HandlerContext & ctx)
 
     default:
         // Mark unrecognized command as UnsupportedCommand
-        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::UnsupportedCommand);
+        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Protocols::InteractionModel::Status::UnsupportedCommand);
```

Comment:
Fwiw, the old setup with `using chip::Protocols::InteractionModel::Status` was probably more readable, but OK.